docs/configuring-exim-relay.md

```diff
@@ -41,9 +41,33 @@ exim_relay_sender_address: "example@{{ exim_relay_hostname }}"
 ########################################################################
 ```
 
-### Relaying email through another SMTP server
+### Enable DKIM support (optional)
+
+By default, exim-relay attempts to deliver emails directly. This may or may not work, depending on your domain configuration.
+
+To improve email deliverability, you can set up *DomainKeys Identified Mail (DKIM)* authentication method, along with SPF and DMARC. Without setting either of them, your outgoing email is most likely to be quarantined as spam.
+
+Exim-relay supports DKIM. With DKIM enabled, exim-relay adds cryptographically signed digital signatures to outgoing emails' headers using the sender's private key.
+
+To enable DKIM support, at first you need to create a DKIM key pair. You can use DKIM with RSA signatures and Ed25519 elliptic curve signatures. For details about setting Ed25519 signatures (its cheacteristics and how to create Ed25519 key pair, etc.), you can refer an article such as [this one](https://www.mailhardener.com/kb/how-to-use-dkim-with-ed25519).
+
+After creating a key pair, add its **public key** to your domain's DKIM DNS record. Look on the internet for a guide about how to do so. Note that every DKIM record must have a unique identifier called as "selector". If you set `eximrelay` as a selector, your DKIM record's DNS name would be `eximrelay._domainkey.example.com`.
```

Comment:
The DKIM selector is mentioned, but it's not mentioned:

- what the the default value (`default`) is, so people can configure DNS correctly
- how to change the default DKIM selector (perhaps just mentioning the new variable is enough)

Or do you intend to make this improvement in another commit later?

Reply:
I'll try to address it with this PR testing with PR testing it for my instance!